templates/definition/charger/homeassistant-switch.yaml

```diff
@@ -27,10 +27,10 @@ params:
     deprecated: true
   - name: switch
     description:
-      de: Entity ID des schaltbaren Geräts
-      en: Entity ID of the switch device
-    service: homeassistant/entities?uri={uri}&domain=switch
-    example: switch.smartsocket
+      de: Entity ID des steuerbaren Geräts
+      en: Entity ID of the controllable device
+    service: homeassistant/entities?uri={uri}&domain=switch,climate,input_boolean
+    example: switch.smartsocket, climate.smartheater, input_boolean.smartplug
```

Comment:
An `input_boolean.smartplug` doesn't sound like something that can be switched? If it is we can add it in a separate PR.